assets/src/pages/webrtcInternals/InternalsSection.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Details from "./Details";
+import Chart from "./Chart";
+import { Section } from "./parsIncomingStats";
+
+type InternalsSectionProps = {
+  section: Section;
+  title: string;
+  isOpen: (key: string) => boolean;
+  toggle: (key: string) => void;
+};
+
+const InternalsSection = ({ title, section, isOpen, toggle }: InternalsSectionProps) => {
+  const { key, descriptive, charts, subsections } = section;
+
+  return (
+    <Details summaryText={title} isOpen={isOpen(key)} toggle={() => toggle(key)}>
```

Comment:
I removed your `useDetailsToggle();` and replaced it with simple `useToggle` and it seems like everything is still working properly. Do I miss something?

```suggestion
type InternalsSectionProps = {
  section: Section;
  title: string;
};

const InternalsSection = ({ title, section }: InternalsSectionProps) => {
  const { descriptive, charts, subsections } = section;
  const [isOpen, toggle] = useToggle(false);

  return (
    <Details summaryText={title} isOpen={isOpen} toggle={toggle}>
```

Reply:
[As mentioned above](https://github.com/membraneframework/membrane_videoroom/pull/163#discussion_r1177355560)